src/tests/tracing/eventpipe/processinfo/processinfo.cs

```diff
@@ -0,0 +1,201 @@
+            var processInfoMessage = new IpcMessage(0x04, 0x00);
+            Logger.logger.Log($"Wrote: {processInfoMessage}");
+            IpcMessage response = IpcClient.SendMessage(stream, processInfoMessage);
+            Logger.logger.Log($"Received: {response}");
+
+            Utils.Assert(response.Header.CommandSet == 0xFF, $"Response must have Server command set. Expected: 0xFF, Received: 0x{response.Header.CommandSet:X2}"); // server
+            Utils.Assert(response.Header.CommandId == 0x00, $"Response must have OK command id. Expected: 0x00, Received: 0x{response.Header.CommandId:X2}"); // OK
+
+            // Parse payload
+            // uint64_t ProcessId;
+            // GUID RuntimeCookie;
+            // LPCWSTR CommandLine;
+            // LPCWSTR OS;
+            // LPCWSTR Arch;
+
+            int totalSize = response.Payload.Length;
+            Logger.logger.Log($"Total size of Payload == {totalSize} b");
+
+            // VALIDATE PID
+            int start = 0;
+            int end = start + 8 /* sizeof(uint63_t) */;
+            UInt64 processId = BitConverter.ToUInt64(response.Payload[start..end]);
+            Utils.Assert((int)processId == pid, $"PID in process info must match. Expected: {pid}, Received: {processId}");
+            Logger.logger.Log($"pid: {processId}");
+
+            // VALIDATE RUNTIME COOKIE
+            start = end;
+            end = start + 16 /* sizeof(GUID) */;
+            Guid runtimeCookie = new Guid(response.Payload[start..end]);
+            Logger.logger.Log($"runtimeCookie: {runtimeCookie}");
+
+            // VALIDATE COMMAND LINE
+            start = end;
+            end = start + 4 /* sizeof(uint32_t) */;
+            UInt32 commandLineLength = BitConverter.ToUInt32(response.Payload[start..end]);
+            Logger.logger.Log($"commandLineLength: {commandLineLength}");
+
+            start = end;
+            end = start + ((int)commandLineLength * sizeof(char));
+            Utils.Assert(end <= totalSize, $"String end can't exceed payload size. Expected: <{totalSize}, Received: {end} (decoded length: {commandLineLength})");
+            Logger.logger.Log($"commandLine bytes: [ {response.Payload[start..end].Select(b => b.ToString("X2") + " ").Aggregate(string.Concat)}]");
+            string commandLine = System.Text.Encoding.Unicode.GetString(response.Payload[start..end]).TrimEnd('\0');
+            Logger.logger.Log($"commandLine: \"{commandLine}\"");
+
+            // The following logic is tailored to this specific test where the cmdline _should_ look like the following:
+            // /path/to/corerun /path/to/processinfo.dll
+            // or
+            // "C:\path\to\CoreRun.exe" C:\path\to\processinfo.dll
+            string currentProcessCommandLine = $"{currentProcess.MainModule.FileName} {System.Reflection.Assembly.GetExecutingAssembly().Location}";
+            string receivedCommandLine = NormalizeCommandLine(commandLine);
```

Comment:
Might want to normalize the strings to lowercase before comparing. I think this is causing 4 of the CI leg failures :-)

Reply:
I recommend we do the minimum normalizing that allows the test to run robustly in CI. For example if the spec says we are supposed to get `"foo.exe" arg` then its better if the test doesn't also accept:
`foo.exe arg`
`"foo.exe arg"`
`'foo.exe arg'`

It also saves a bunch of complexity in the test... Rather than a NormalizeCommandLine() function what about:
````
string expectedCommandLine = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ?
    $"\"{currentProcess.MainModule.FileName}\" {System.Reflection.Assembly.GetExecutingAssembly().Location}" : 
    $"{currentProcess.MainModule.FileName} {System.Reflection.Assembly.GetExecutingAssembly().Location}";
Utils.Assert(expectedCommandLine.Equals(commandLine, StringComparison.OrdinalIgnoreCase), ...)
````
````